### PR TITLE
Tmpfiles fix precedence with plus sign

### PR DESCRIPTION
**Pull request by fbuihuu**

### CODE REVIEW

**Comment by bluca** [comment]
```
TEST-22-TMPFILES:                   FAIL     (  5 s)
```

File: src/tmpfiles/tmpfiles.c [hunk 1]
```diff
@@ -2896,6 +2896,26 @@ static int parse_age_by_from_arg(const char *age_by_str, Item *item) {
...
+        for (size_t n = 0; n < existing->n_items; n++) {
+                Item *e = existing->items + n;
+
+                if (item_compatible(e, i))
+                        continue;
+
+                /* Only multiple 'w+' lines for the same path are allowed. */
+                if (e->type != WRITE_FILE || !e->append_or_force ||
+                    i->type != WRITE_FILE || !i->append_or_force)
+                    return true;
```
Comment (yuwata) [inline src/tmpfiles/tmpfiles.c]: The condition must be inverted.
```
if (e->type == WRITE_FILE && e->append_or_force &&
    i->type == WRITE_FILE && i->append_or_force)
        continue;

return true;
```

BTW, why not to add the condition into `item_compatible()`?

Comment (fbuihuu) [reply src/tmpfiles/tmpfiles.c]: Because It looked more readable but it's subjective so I've no strong argument.

Comment (yuwata) [reply src/tmpfiles/tmpfiles.c]: Hm, wait, my comment was not correct. Sorry. It seems your code is fine.

But please fix the indentation.

**Review by bluca — APPROVED** [review]

**Review by yuwata — APPROVED** [review]

**Pull request merged**